AdyenNetworking/APIClient/HTTPResponse.swift

```diff
@@ -6,7 +6,7 @@
 
 import Foundation
 
-public struct HTTPResponse<R: Response> {
+public struct HTTPResponse<R: Response>: Sendable {
```

Comment:
Can this backfire us?

> To satisfy the requirements of the `Sendable` protocol, an enumeration or structure must have only sendable members and associated values.

So now all model structs/classes from all dependent SDKs have to be marked "Sendable" ?

Reply:
Yes, right now not conforming to `Sendable` causes warnings. It will causes errors starting with Swift 6